Blackjack/blackjack.py

```diff
@@ -0,0 +1,220 @@
+# IMPORT MODULES AND DEFINE VARIABLES
+
+import random
+
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
```

Comment:
make the suits and ranks as class variables of Card class and values in Hand class

Reply:
Done!